### ISSUE TITLE
Missing License

### ISSUE THREAD

**Issue by tudalex** [rosedu/site#55]
Missing a LICENSE file in the root repository folder that specifies the license of the website.

**Comment by maria** [comment]
I added the following license: https://github.com/rosedu/site/blob/master/LICENSE.md. If someone is against it or the License selected, please state your reason in a week's time.

cc-ing all the contributors @alexef @razvand @mariuscoto @flaviusanton @camelia-groza @tudalex @dserban  @alrra @mgax @mihaimaruseac @alexandrujuncu @mihneadb @lauuuuuura @vilie @silviupopescu @vcarbune @adriansc @ddvlad

**Comment by dserban** [comment]
The image files below are licensed in a way that is not compatible with the license you are suggesting.
```
resources/promo/webdev_promo_banner.png
activities/images/activity_logo_webdev.png
```

**Comment by lauuuuuura** [comment]
@maria : why not [BSD](https://www.facebook.com/rosedu.org/photos/a.111658015562907.14555.111655162229859/1012882262107140/?type=3&permPage=1)?

**Comment by maria** [comment]
@dserban so you are referring to 
  - https://github.com/rosedu/site/blob/master/activities/images/activity_logo_webdev.png
 - https://github.com/rosedu/site/blob/master/resources/promo/webdev_promo_banner.png

OK, we will remove them and replace them with our artwork. thank you

**Comment by maria** [comment]
@lauuuuuura I have to admit I didn't look into it. Here are more details http://choosealicense.com/licenses/ - I wanted a license more explicit and which defined that the work one does for the project will fall under the project license, due to the problems we had in the past. :) that makes sense?

**Comment by tudalex** [comment]
@maria if you want to be explicit about this, the only way of doing it is with a contributor license agreement (like the one you have to sign for contributing to Google OpenSource projects like Chromium or Android[0]) in which the contributor grants all copyrights to the Rosedu Association before working on a project.
This does not replace the usual BSD/GPL license, but complements it, because in this way there is only one author, ROSEdu, and if you later need to change something you don't have to get the agreement from everybody who worked on the project.

[0] - https://cla.developers.google.com/

**Comment by maria** [comment]
@tudalex :+1: Let's do this. Great idea

**Comment by mgax** [comment]
I also prefer a MIT/BSD license. Also, if you intend to relicense a codebase, you need explicit agreement from all contributors with significant contributions. Failing to answer within a week is not the same as explicit agreement.

**Comment by tudalex** [comment]
:+1: @mgax
Not just significant, but all contributors! Also a contributor doesn't have to have a reason why he doesn't agree to a license. And if you want to 

I just right now realised that this is not a LICENSE proposal waiting to be merged, but was already merged! This is not nice! Just saying.

**Comment by maria** [comment]
@mgax @tudalex fair enough. I updated the [License.md file](https://github.com/rosedu/site/blob/master/LICENSE.md), we will open a PR after we discuss. 

I cc-ed every contributor here -  https://github.com/rosedu/site/issues/55#issuecomment-174334589. The week time is   from a model we have in the Romanian legal department, where a decision can be questioned in a week time otherwise is accepted. This idea is from a previous discussion we had on emails between the community members. And I think is helpful to not prolong this discussion indefinitely.

So, we have 3 votes for a BSD type of license, I think for projects like our websites we should have a more restrictive license and for other projects a BSD type (ex: rosedu challenge) and a CLA like @tudalex suggested.

**Comment by mgax** [comment]
Come to think of it, we're licensing content here so a creative commons licence is more appropriate. I propose CC-BY, again the most liberal, there's no reason to restrict people from using our text; they have to provide attribution anyway.

**Comment by maria** [comment]
@mgax is not only text it's a bit of code also. I think BSD (it's seems is the most popular) and we will create a CLA in the near future.

That sounds ok? There are still 2 days in the timeframe I suggested to get this done, so waiting for other feedback.

**Comment by maria** [comment]
This is the PR https://github.com/rosedu/site/pull/70

**Comment by mgax** [comment]
@maria did we ever get licensing agreement from all contributors? E.g. I don't see @alexef or @razvand  agreeing, and GitHub says they are the biggest contributors. And what's the point of making a pull request if you're going to approve it yourself within a minute?

**Comment by tudalex** [comment]
@maria I totally agree with @mgax I don't see the point of making a pull request as a proposal, just for it to be merged 55 seconds later....

I think the only nice way of doing this is that everybody leaves a comment with the a text similar to "I agree to a BSD license for my contributions." What do you think @mgax ? So instead of fighting let's talk to everybody and convince them to reply to this issue.

So I'll start:
I agree to a BSD license for my contributions.

**Comment by lauuuuuura** [comment]
I agree to a BSD license for my contributions.

**Comment by maria** [comment]
Sorry for the "abusive" way of managing this issue. I reverted my PR, you can find it here https://github.com/rosedu/site/pull/71.

Let's see if all  @alexef @razvand @mariuscoto @flaviusanton @camelia-groza @dserban @alrra @mihaimaruseac @alexandrujuncu @mihneadb @vilie @silviupopescu @vcarbune @adriansc @ddvlad of use agree on using a License and the one here https://github.com/rosedu/site/pull/71 

I one agree to a BSD license for all my contributions made to this repository.

**Comment by mihaimaruseac** [comment]
BSD is fine :)

**Comment by ddvlad** [comment]
I'm surprised to find I've contributed to this repo :)

I'm fine with BSD as well.

**Comment by vilie** [comment]
I agree to a BSD license for my contributions.

**Comment by mgax** [comment]
BSD :+1:

**Comment by mariuscoto** [comment]
I agree to a BSD license for my contributions.

**Comment by mihneadb** [comment]
I agree to a BSD license for my contributions

-- 
--
Mihnea Dobrescu-Balaur
Sent from a mobile device.

**Comment by alexef** [comment]
I agree to a BSD license for my contributions

**Comment by alexandrujuncu** [comment]
I think this discussion is pointless for the site itself. It's one of the repos less likely to be forked.

Probably even WTFPL would be ok.
To not complicate things, I agree to a BSD license for my contributions.
